README.md

```diff
@@ -37,12 +37,12 @@ library.
 
 On Redhat/Fedora/CentOS systems
 ```sh
-$ yum install gcc gcc-c++ autoconf automake libtool pkgconfig flex bison readline-devel ncurses5-devel lua-devel
+$ yum install gcc gcc-c++ autoconf automake libtool pkgconfig flex bison readline-devel ncurses5-devel lua-devel llvm-toolset-7
 ```
 
 On Debian/Ubuntu and derived systems
 ```sh
-$ apt-get install build-essential automake libtool pkg-config flex bison libreadline-dev libncurses5-dev liblua5.1-0-dev
+$ apt-get install build-essential automake libtool pkg-config flex bison libreadline-dev libncurses5-dev liblua5.1-0-dev clang-format
```

Comment:
I am running a mix of packages from debian stable and testing, but still only got clang-format-3.8.1 which fails to parse several of the options in the clang-format config file.

Reply:
I'm not sure how to handle this. I believe that the included `.clang-format` should be supported from clang-format 4.0.0 and on.